core/build.gradle

```diff
@@ -1,4 +1,6 @@
 dependencies {
+    compile project(':docs-client')
```

Comment:
This has to be removed because otherwise the pom will refer to an unpublished artifact.

Reply:
Completely forgot to remove this! It's not needed since `builtBy` takes care of the task dependency.